src/include/os.h

```diff
@@ -109,11 +109,17 @@ struct __wt_fh {
 	const char *name;			/* File name */
 
 	uint64_t name_hash;			/* hash of name */
+	uint64_t last_sync;			/* time of background fsync */
+	volatile uint64_t written;		/* written since fsync */
```

Comment:
This field is now, essentially, a duplicate of the `block->os_cache_dirty` field that is used with `os_cache_dirty_max`. It would be good to remove that redundancy. This is a thread that is coming in through the file handles, so it isn't using the block manager.

Reply:
This issue is now in WT-4547.